tictactoe/test/utilities_tests.erl

```diff
@@ -0,0 +1,115 @@
+-module(utilities_tests).
+-include_lib("eunit/include/eunit.hrl").
+
+mark_board_test() ->
+  [{  "returns board with marker placed",
+      ?assertEqual(
+        ["X",2,3,4,5,6,7,8,9],
+        utilities:mark_board(1, [1,2,3,4,5,6,7,8,9], "X")),
+      ?assertEqual(
+        ["O",2,3,4,5,6,7,8,9],
+        utilities:mark_board(1, [1,2,3,4,5,6,7,8,9], "O"))}].
+
+change_player_test() ->
+  [{  "changes player marker",
+      ?assertEqual(o,utilities:change_player(x)),
+      ?assertEqual(x,utilities:change_player(o))}].
+
+validate_move_test() ->
+  [{  "validates a move",
+      ?assertEqual( invalid,
+        utilities:validate_move_position(1,[o,2,3,4,5,6,7,8,9])),
+      ?assertEqual( invalid,
+        utilities:validate_move_position(10,[1,2,3,4,5,6,7,8,9])),
+      ?assertEqual( {valid, 1},
+        utilities:validate_move_position(1,[1,2,3,4,5,6,7,8,9])),
+      ?assertEqual( invalid,
+        utilities:validate_move_position(0,[1,2,3,4,5,6,7,8,9]))}].
+
+board_full_test() -> 
+  [{  "checks if the game is over",
```

Comment:
This is kind of weird naming. If you are checking if the game is over, and your last condition is a potential tie, maybe this method should be more reflective of that. You have a `game_won` method, maybe there should be a `game_tied` method that evaluates that the game is not won and that the board is full.

Reply:
I mostly agree and will rename to `check_for_draw`... 
The reason I say mostly agree is that
1. `game_tied` to me expresses the game is still running and both players have equal chance of winning  
2. there's no need for this method to call the game_won function since this function is only evaluated if the game_won function is false in the first place, it may be a flaw in my control flow design but for now I'll rename rather than alter the design